templates/password-warning.spt

```diff
@@ -8,13 +8,13 @@
         <h3 class="text-danger">{{ _("This new password is not safe") }}</h3>
         <p>{{
             _(
-                "We have detected that your password is a commonly used one. It appears many "
-                "times in various leaked databases, which makes it very insecure."
+                "Your password is a commonly used one, appearing many "
+                "times in various leaked databases, making it very insecure."
             ) if password_status == 'common' else _(
-                "We have detected that your password has been compromised: it appears in one "
+                "Your password has been compromised: It appears in one "
```

Comment:
Same as above.
The reason for : Bigversal is to make for fewer errors in translation.
Other languages require : Bigversal, whereas it is optional in English.